### DISCUSSION TITLE
Easy way to determine what the current LTS is?

### DISCUSSION THREAD

**Discussion by milesfryhover** [openssl/openssl#25031] [General Discussion]
I'm trying to gather information for all releases including their EOL dates however I'm having a hard time finding where a version is defined as a LTS. Can anyone point me in the right direction? 

For example, will an LTS always be the release incremented by the major release? So next LTS will be 4.0? Or is there a page/document where it indicated each version and whether or not it's an LTS?

**Comment by kulkarniamit** [comment]
These links should help you answer the questions about LTS: 
* https://openssl-library.org/policies/releasestrat/index.html (Release Strategy)
* https://openssl-library.org/policies/general/versioning-policy/ (OpenSSL release versioning policy)
* https://openssl-library.org/source/index.html (EOL dates of versions)
* [Which version will be the next LTS release of OpenSSL?](https://github.com/openssl/openssl/discussions/23735)

**Reply by milesfryhover** [reply]
So I should assume if there isn't further definition of what an LTS can be then it's up in the air? That documentation only really provides that: 
- A LTS will be announced every 4 years
- A LTS will be supported for 5 years

No indication on how/where it will be announced/indicated or if it will also come with a new major number.

**Reply by kulkarniamit** [reply]
According to the policy we can expect the next LTS to be announced by 7th September 2025.
Here's some more information about the possible major and minor numbers: https://github.com/openssl/openssl/discussions/23674#discussioncomment-8581050
> Under our policy we must designate a new LTS every 4 years (at a maximum). 3.0 was released on 2021-09-07 - so we must designate a new LTS by 2025-09-07 at the latest. Since we now release every April and October one of the releases between now and the one in April 2025 will be the next LTS.

The announcement should be available on the [newslog](https://openssl-library.org/news/newslog/index.html) or [blog posts](https://openssl-library.org/post/).

**Reply by paulidale** [reply]
It would be nice if the EOL dates could be included in the table of current releases...

**Reply by milesfryhover** [reply]
> It would be nice if the EOL dates could be included in the table of current releases...

Agreed. It would also be nice if there was an indicator on which of the releases were LTS. As it is, you have to scour multiple pages for information on it.

**Reply by t8m** [reply]
LTS is not necessarily associated with a new major release. Most likely the next LTS will be the 3.5 version. As we now have time based release process with releases going out every 6 months in April and October we will likely designate LTS to be every xth release which will make the release cycle fully predictable. We did not decide on the number _x_ yet though.
